export type SaveExceptionSuccess = {
    result: "exception" | "ok"
    exception_id: number
}
